frontend/api/index.js

```diff
@@ -111,14 +111,86 @@ app.get('/config/project-overrides', (req, res) => {
+                            Ld;else if(!Ld.syntheticConfig){gf=Ld;break b}}Ld||(Ld=Db);if(0<rg){var bh=rg+Cf.length+5;Ld.app=Ud.length>bh?Ud.substring(bh):"Default%20Application"}else if(Ud){var sg=se.exec(Ud);sg&&(Ld.app=sg[1])}gf=Bf}else gf=ke}if(gf)for(var Rf in gf)Object.prototype.hasOwnProperty.call(gf,Rf)&&(ee=Rf,Db[ee]=gf[ee]);var Sf=ba();try{var fe=(gf=de)&&gf.getItem(Sf);if(fe){var Tf=E(fe),Te=R(Tf.config||""),qh=Db.lastModification||"0",Nh=ab((Te.lastModification||Tf.lastModification||"0").substring(0,13)),Ni="string"===
+                    typeof qh?ab(qh.substring(0,13)):qh;if(!qh||Nh>=Ni)if(Te.csu=Tf.name||Ga("csu"),Te.featureHash=Tf.featureHash||Ga("featureHash"),Te.agentUri&&yc(Te),zc(Te,!0),wb(Te),zb(Te),Nh>(+Db.lastModification||0)){var rh=W("auto"),hd=W("legacy");Db=Eb(Te);Db.auto=rh?"1":"0";Db.legacy=hd?"1":"0"}}}catch(Mh){}zc(Db);try{var tg=Db.ign;if(tg&&(new RegExp(tg)).test(Wa.location.href)){document.dT_=Wa.dT_=void 0;var He=!1;break a}}catch(Mh){}if(K()){var Df=navigator.userAgent,Ue=Df.lastIndexOf("RuxitSynthetic");if(-1===
+                        Ue)var fg={};else{var Ve=Df.substring(Ue+14);if(-1===Sa(Ve," c"))fg={};else{Sf={};fe=0;for(var te=Ve.split(" ");fe<te.length;fe++){var ch=te[fe];if("c"===ch.charAt(0)){var gg=ch.substring(1),Ie=gg.indexOf("="),ug=gg.substring(0,Ie),Mg=gg.substring(Ie+1);ug&&Mg&&(Sf[ug]=Mg)}}fg=Sf}}Ve=void 0;for(Ve in fg)Object.prototype.hasOwnProperty.call(fg,Ve)&&fg[Ve]&&(Db[Ve]=fg[Ve]);Eb(Db)}He=!0}if(!He)throw Error("Error during config initialization");Pe();Lg=Wa.dT_.disabled;var hf;if(!(hf=Ga("agentLocation")))a:{var Oh=
+                    Ga("agentUri");if(Oh||document.currentScript){var jf=Oh||document.currentScript.src;if(jf){He=jf;var Ph=-1===Sa(He,"_bs")&&-1===Sa(He,"_bootstrap")&&-1===Sa(He,"_complete")?1:2,Ng=jf.lastIndexOf("/");for(He=0;He<Ph&&-1!==Ng;He++)jf=jf.substring(0,Ng),Ng=jf.lastIndexOf("/");hf=jf;break a}}var dh=location.pathname;hf=dh.substring(0,dh.lastIndexOf("/"))}yf=hf;je=Ga("csu")||"ruxitagentjs";"true"===Za("dtUseDebugAgent")&&0>je.indexOf("dbg")&&(je=Ga("debugName")||je+"dbg");if(!W("auto")&&!W("legacy")&&
+                    !Lg){var Ef=Ga("agentUri")||ad(Ga("featureHash"));if(W("async")||"complete"===document.readyState)Zc(Ef,W("async"),void 0,void 0,"dtjsagent");else{var Oi="".concat("dtjsagent","dw");document.write('<script id="'.concat(Oi,'" type="text/javascript" src="').concat(Ef,'">\x3c/script>'));document.getElementById(Oi)||Zc(Ef,W("async"),void 0,void 0,"dtjsagent")}}Za("dtCookie")&&Cb("cooO",!0);zf();Cb("pVO",!!nb("dt-pVO"));Wb(fc);cg=18E5;Gg=oa("hvt")||216E5;jb(Fd,[1])}catch(Mh){delete Wa.dT_,fb()&&Wa.console.log("JsAgent initCode initialization failed!",
+                Mh)}}})()})();`
+  }
   res.setHeader('Cache-Control', 's-max-age=1, stale-while-revalidate')
   res.setHeader('content-type', 'application/javascript')
   res.send(`window.projectOverrides = {
         ${output}
     };
+    
+    ${dynatrace}
```

Comment:
Should this be included in the `res.send()` ?

Reply:
Yeah we want it to return the dynatrace script